content/en/blog/2024/inpod-traffic-redirection-ambient/index.md

```diff
@@ -0,0 +1,235 @@
+and [Docker Desktop](https://github.com/istio/istio/issues/47436), with CNI implementations like [Cilium](https://github.com/istio/istio/issues/44198) and [Calico](https://github.com/istio/istio/issues/40973),
+and on services that ship in-house CNI implementations
+like [OpenShift](https://github.com/istio/istio/issues/42341) and [Amazon EKS](https://github.com/istio/istio/issues/42340).
+Getting broad support for Kubernetes anywhere has become the No. 1 requirement for ambient mesh moving to Beta — people have come to expect Istio to
+work on any Kubernetes platform and with any CNI implementation. After all, ambient wouldn’t be ambient without being all around you!
+
+At Solo, we've been integrating ambient mode into our Gloo Mesh product, and came up with an innovative solution to this problem.
+We decided to [upstream](https://github.com/istio/istio/issues/48212) our changes in late 2023 to help ambient reach beta faster,
+so more users can operate ambient in Istio 1.21 or newer, and enjoy the benefits of ambient sidecar-less mesh in their platforms
+regardless of their existing or preferred CNI implementation.
+
+## How did we get here?
+
+### Service meshes and CNIs: it's complicated
+
+Istio is a service mesh, and all service meshes by strict definition are not *CNI implementations* - service meshes require a
+[spec-compliant, primary CNI implementation](https://www.cni.dev/docs/spec/#overview-1) to be present in every Kubernetes cluster, and rest on top of that.
+
+This primary CNI implementation may be provided by your cloud provider (AKS, GKE, and EKS all ship their own), or by third-party CNI
+implementations like Calico and Cilium. Some service meshes may also ship bundled with their own primary CNI implementation, which they
+explicitly require to function.
+
+Basically, before you can do things like secure pod traffic with mTLS and apply high-level authentication and authorization policy at the
+service mesh layer, you must have a functional Kubernetes cluster with a functional CNI implementation, to make sure the basic networking
+pathways are set up so that packets can get from one pod to another (and from one node to another) in your cluster.
+
+Though some service meshes may also ship and require their own in-house primary CNI implementation, and it is sometimes possible to run two
+primary CNI implementations in parallel within the same cluster (for instance, one shipped by the cloud provider, and a 3rd-party
+implementation), in practice this introduces a whole host of compatibility issues, strange behaviors, reduced feature sets, and some
+incompatibilities due to the wildly varying mechanisms each CNI implementation might employ internally.
+
+To avoid this, the Istio project has chosen not to ship or require their own primary CNI implementation, or even require a "preferred" CNI
```

Comment:
```suggestion
To avoid this, the Istio project has chosen not to ship or require our own primary CNI implementation, or even require a "preferred" CNI
```